src/main/java/com/google/sps/data/AvailabilityTimeSlots.java

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.TextStyle;
+import java.util.Locale;
+import java.time.format.DateTimeFormatter;
+
+// TODO: Add a test for the constructor.
+
+/**
+ * A collection of AvailabilityTimeSlot Objects, which will eventually be generated with information
+ * from datastore.
+ */
+public class AvailabilityTimeSlots {
```

Comment:
The public functions for this class are just

```java
  public AvailabilityTimeSlots(String timeZoneOffsetString)
  public List<AvailabilityTimeSlot> getTimeSlots();
```

All this is is a fancy fake data generator for a List<AvailabilityTimeSlot>. Perhaps rename it to match, which will free this name for a proper AvailabilityTimeSlots class with domain specific logic that is independent of this fake data generation in the future.

`class FakeAvailabilityTimeSlotGenerator`.

Reply:
This data isn't fake though. Eventually the selectedStatuses will be generated based on what availability time slots are already in datastore.

Reply:
I'm sort of implicitly requesting that you separate the logic responsible for loading the data from the representation of that data. This represents a better [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).

Consider a test depending on some AvailabilityTimeSlots. Is it easier for the test to use some locally defined AvailabilityTimeSlots (which, is really just data), or for you to load the data into DataStore, then create a RelevantDao, then create an "AvailabilityTimeSlots" which loads the fake data from the RelevantDao from the DataStore? 

I think the representation of the time slots can be easily decoupled from how that data is loaded, and in fact you've already done it. The last step is just to rename this class to AvailabilityTimeSlotLoader or something..

Reply:
I renamed the class to AvailabilityTimeSlotGenerator.